src/main/kotlin/Server.kt

```diff
@@ -31,6 +33,8 @@ class Server(private val serverSocket: ServerSocket,
                 writeResponse(response)
             } catch (e: Exception) {
                 println("Something went wrong: ${e.message}")
+                val response = serverError.handleError(responseBuilder)
```

Comment:
since you have the error message, do you think that should be included in the response? Or do you think exposing unpredictable error messages a security vulnerability?

Reply:
My decision to not include the error message in the response wasn't motivated by a security point of view, but more along the lines of the client not needing to know about the internal workings of the server. Additionally, the client can't do much with that information because it is out of their control. However, thank you for bringing security vulnerability up. I did some reading and if the client is a malicious user then they could use this information to attack the service. I don't think providing just the exception message would be enough to expose the service to such a vulnerability. Including a stack trace; however, could provide the attacker with more information.